buildSrc/build.gradle

```diff
@@ -112,6 +112,8 @@ repositories {
 dependencies {
   compile localGroovy()
   
+  compile "commons-codec:commons-codec:${props.getProperty('commonscodec')}"
```

Comment:
We can add the version here directly the props file is more for dependencies used in multiple projects.